kedro/framework/cli/project.py

```diff
@@ -353,6 +355,7 @@ def activate_nbstripout(
     callback=_reformat_load_versions,
 )
 @click.option("--pipeline", "-p", type=str, default=None, help=PIPELINE_ARG_HELP)
+@click.option("--namespace", "-ns", type=str, default=None, help=NAMESPACE_ARG_HELP)
```

Comment:
To keep the ordering of `click.option` decorators consistent with the arguments in the function signature, I think this should be moved to the bottom.

Reply:
Do we actually follow the order? The first `click` option is `from_input` but the first argument would be `tag`. I think it only affects when you do `kedro run -h`, it will follow the order that how you decorate your function. In general, I think it should follow the order of importance. I would expect `pipeline` to be the first argument in that case.

I don't have a strong opinion about moving it to the back, the only reason I keep it next to `pipeline` is that I feel these two arguments are in the same category. Similarly, the `nodes` arguments are together, and the config-related arguments are grouped as well. @merelcht or @AntonyMilneQB Is this something we've discussed in the past?

Reply:
Fully agree with you here @noklam 👍  We should go for what makes sense to the user when you do `kedro run --help`; no need to be consistent with the order of `KedroSession.run` (which as you've said is somewhat arbitrary).